### PR TITLE
Fix typos and suggest nginx -t before restart

### PR DESCRIPTION
**Pull request by Pablohn26**

### CODE REVIEW

File: setup-production.adoc [hunk 1]
```diff
@@ -353,6 +353,7 @@ cd taiga-events
 .Install all the javascript dependencies needed
 [source,bash]
 ----
+sudo apt-get -y npm
```
Comment (superalex) [inline setup-production.adoc]: Hello!,

npm should be installed yet in the line: sudo apt-get install -y nodejs npm

Could you please remove this one?

**Comment by superalex** [comment]
Hello @Pablohn26 

Thank you very much for the PR, I've added just a small comment on it, could you please check it?

Regards!,

File: setup-production.adoc (hunk 1 above)
Comment (Pablohn26) [inline setup-production.adoc]: Where is that line?

Comment (superalex) [inline setup-production.adoc]: Sorry, I was checking the setup-development.adoc (in that file it's included)

**Comment by superalex** [comment]
Thank you very much! :+1:

**Pull request merged**